src/test/java/org/broadinstitute/hellbender/tools/spark/pathseq/loggers/PSFilterFileLoggerTest.java

```diff
@@ -13,7 +13,7 @@
 import java.util.Collections;
 
 public class PSFilterFileLoggerTest  extends GATKBaseTest {
-
+ 
```

Comment:
This is odd, why you need to do this to force a rebuild? perhaps you need to ask David R and co how to handle it in travis appropriately. In any case please revert this empty extra line.

Reply:
Done